packages/browser-wallet/src/channel-wallet-types.ts

```diff
@@ -0,0 +1,27 @@
+import {DirectFunder, SharedObjective, SignedState} from '@statechannels/wallet-core';
+import {Interpreter} from 'xstate';
+
+export type AnyInterpreter = Interpreter<any, any, any>;
+export type OnObjectiveEvent = (event: DirectFunder.OpenChannelEvent) => void;
```

Comment:
The name`OnObjectiveEvent` seems to indicate an event handler but it's actually used by the UI to trigger an objective event. Could we name it something like `triggerObjectiveEvent` ?

Reply:
Good call. Addressed in 9141a23